code/modules/hydroponics/grown.dm

```diff
@@ -45,6 +47,9 @@
 		seed.prepare_result(src)
 		transform *= TRANSFORM_USING_VARIABLE(seed.potency, 100) + 0.5 //Makes the resulting produce's sprite larger or smaller based on potency!
 		add_juice()
+	
+	if(discovery_points && discovery_points > 0)
```

Comment:
```suggestion
	if(discovery_points)
```

Reply:
if([thing]) 
if [thing] isnt null (ie: set to a positive integer), it will always be true.

Reply:
Ah that makes sense, I will make the change to both instances when I get home